src/views/Game/Game.tsx

```diff
@@ -1663,20 +1701,24 @@ export function Game(): JSX.Element | null {
                         <div className={"right-col" + (experimental ? " experimental" : "")}>
                             {(zen_mode || null) && <div className="align-col-start"></div>}
                             {(view_mode === "square" || view_mode === "wide" || null) && (
-                                <PlayerCards
-                                    historical_black={historical_black}
-                                    historical_white={historical_white}
-                                    estimating_score={estimating_score}
-                                    zen_mode={zen_mode}
-                                    black_flags={black_flags}
-                                    white_flags={white_flags}
-                                    black_ai_suspected={bot_detection_results?.ai_suspected.includes(
-                                        historical_black?.id,
-                                    )}
-                                    white_ai_suspected={bot_detection_results?.ai_suspected.includes(
-                                        historical_white?.id,
-                                    )}
-                                />
+                                <>
+                                    <PlayerCards
+                                        historical_black={historical_black}
+                                        historical_white={historical_white}
+                                        estimating_score={estimating_score}
+                                        zen_mode={zen_mode}
+                                        black_flags={black_flags}
+                                        white_flags={white_flags}
+                                        black_ai_suspected={bot_detection_results?.ai_suspected.includes(
+                                            historical_black?.id,
+                                        )}
+                                        white_ai_suspected={bot_detection_results?.ai_suspected.includes(
+                                            historical_white?.id,
+                                        )}
+                                    />
+                                    {frag_game_information()}
+                                    {frag_rengo_header()}
+                                </>
```

Comment:
Maybe, this (and the one above for portrait mode) should be a `<div>` (not a `<>`) to make the player cards and the condensed-game-info indivisible? I only have a vague idea about how layout works, but maybe the condensed-game-info is allowed to float down below the "White's move" stuff, and sometimes that's happening.

Reply:
163956d defensively does this. I don't fully understand if/why it's necessary, but...